scuba/config.py

```diff
@@ -155,9 +155,12 @@ def override(self, node: yaml.nodes.Node) -> OverrideMixin:
         assert isinstance(obj, OverrideMixin)
         return obj
 
+    def not_yet_implemented(self, node: yaml.nodes.Node):
```

Comment:
Let's name this `!reference`, following the existing pattern. That it is not yet implemented is mostly inconsequential.